File: src/Generated20260123.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260123 {
public:
    // Generated class 20260123_064127_796403
    int value = 802;
    string created = "20260123_064127_796403";
    
    int process() {
        int result = value;
        for (int i = 0; i < 20; i++) {
            result += i * 5;
        }
        return result;
    }
};

int main() {
    Generated20260123 obj;
    cout << obj.process() << endl;
    return 0;
}

File: src/Generated20260123.go
package main

import "fmt"

type Generated20260123 struct {
    // Generated struct 20260123_064531_341591
    Value int
    Created string
}

func (c *Generated20260123) Process() int {
    result := c.Value
    for i := 0; i < 22; i++ {
        result += i * 5
    }
    return result
}

func main() {
    obj := Generated20260123{Value: 171, Created: "20260123_064531_341591"}
    fmt.Println(obj.Process())
}